test/0.8.9/deposit-security-module.test.js

```diff
@@ -819,10 +871,7 @@ contract('DepositSecurityModule', ([owner, stranger, guardian]) => {
         assert.isTrue((await depositSecurityModule.getGuardians()).includes(GUARDIAN1))
       })
       it(`setGuardianQuorum can't be called by non-admin`, async () => {
-        await assertRevert(
-          depositSecurityModule.setGuardianQuorum(1, { from: stranger }),
-          `NotAnOwner("${stranger}")`
-        )
+        await assertRevert(depositSecurityModule.setGuardianQuorum(1, { from: stranger }), `NotAnOwner("${stranger}")`)
```

Comment:
I'd prefer it not reformatting `assert.revert`s like this but not sure we can configure it that way

Reply:
Probably, we can't do anything here.